app/views/account_reset/request/show.html.erb

```diff
@@ -2,31 +2,29 @@
 
 <%= render PageHeadingComponent.new.with_content(t('account_reset.request.title')) %>
 
-<p class='mt-tiny margin-bottom-0'>
-  <%= t('account_reset.request.info').html_safe %>
-</p>
-<br/>
+<% t('account_reset.request.info').each do |info_p| %>
```

Comment:
I love this use of arrays for translations! better than our `bullet_1` `bullet_2` approach we've used elsewhere

Reply:
> I love this use of arrays for translations! better than our `bullet_1` `bullet_2` approach we've used elsewhere

Yeah, I think so too! Case in point, [these changes](https://github.com/18F/identity-idp/pull/5863/files#diff-23c3048dc1ecc7c2616d1af70a83d241a868cd1207625fb9a51a242e9f5719a5) in #5863 will leave a weird ordering `text1a`, `text2`, `text3` 😬